From b4d4b340fc9beffb41fc5c26b1f0bf616d9d522a Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 20 Jan 2018 17:36:21 +0000
Subject: [PATCH 1/1] log: add category LOGC_EFI

The EFI implementation does not fit into any of the existing categories.

Provide LOGC_EFI so that EFI related message can be filtered.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/README.log | 1 +
 include/log.h  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/doc/README.log b/doc/README.log
index f653fe7d79..be9978a270 100644
--- a/doc/README.log
+++ b/doc/README.log
@@ -51,6 +51,7 @@ The following main categories are defined:
    LOGC_BOARD	- Related to board-specific code
    LOGC_CORE	- Related to core driver-model support
    LOGC_DT	- Related to device tree control
+   LOGC_EFI	- Related to the EFI implementation
 
 
 Enabling logging
diff --git a/include/log.h b/include/log.h
index 8083b64831..b155725de0 100644
--- a/include/log.h
+++ b/include/log.h
@@ -43,6 +43,7 @@ enum log_category_t {
 	LOGC_BOARD,
 	LOGC_CORE,
 	LOGC_DT,
+	LOGC_EFI,
 
 	LOGC_COUNT,
 	LOGC_END,
-- 
2.15.1

